releng/tools/org.openjdk.jmc.util.releasenotes/notes/8.1/notes.xml

```diff
@@ -0,0 +1,219 @@
+			</description>
+			<image>jfr/allocationprof.png</image>
+		</note>
+		<note version="8.1">
+			<title>New Page for Peeking into the Constant Pools</title>
+			<description>
+				There is a new page available for taking a look at what
+				constants are available in the recording. This can, for example, be
+				useful when creating custom events to see where all that storage and
+				memory is being used.
+			</description>
+			<image>jfr/constantpool.png</image>
+		</note>
+		<note version="8.1">
+			<title>Open Recordings with .lz4 extension</title>
+			<description>
+				For convenience, files with the .lz4 extension will now
+				be attempted to be opened as flight recordings. This is since lz4 is
+				a common compression to use with flight recordings.
+			</description>
+			<image>jfr/lz4.png</image>
+		</note>
+	</component>
+	<!-- ******************************* Agent ******************************* -->
+	<component>
+		<name>JMC Agent Plug-in</name>
+		<note version="8.1.0">
+			<title>New JMC Agent Plug-in</title>
+			<description>
+				There is now a new agent plug-in available for JMC,
+				which allows configuring where to emit flight recording
+				events in an
+				already running process.
+			</description>
+			<image>agent/agent.png</image>
+		</note>
+	</component>
+	<!-- ******************************* Bugfixes ****************************************** -->
+	<bugfixes>
+		<bugfix>
+			<area>JFR</area>
+			<bugid>6939</bugid>
+			<synopsis>Time range indicator update problem fixed</synopsis>
+			<description>
+				Sometimes the time range indicator wasn't updated when
+				setting the time
+				range. This is now fixed.
+			</description>
+		</bugfix>
+		<bugfix>
+			<area>JFR</area>
+			<bugid>7007</bugid>
+			<synopsis>
+				Unable to edit run configurations for eclipse project after
+				installing JMC
+				plugin fixed
+			</synopsis>
+			<description>
+				Previously it would not be possible to edit run
+				configuration after installing
+				the experimental JMC launcher plug-in.
+				This has now been resolve.
+			</description>
+		</bugfix>
+	</bugfixes>
+	<!-- ******************************* Known issues ****************************************** -->
+	<knownissues>
+		<issue>
+			<area>General</area>
+			<bugid>4270</bugid>
+			<synopsis>Hibernation and time</synopsis>
+			<description>
+				After the bugfix of
+				https://bugs.openjdk.java.net/browse/JDK-6523160 in JDK 8, the
+				RuntimeMXBean#getUptime() attribute was re-implemented to mean
+				"Elapsed time of JVM process", whilst it previously was implemented
+				as time since start of the JVM process. The uptime attribute is used
+				by JMC, together with RuntimeMXBean#getStartTime(), to estimate the
+				actual server time. This means that time stamps, as well as
+				remaining time for a flight recording, can be wrong for processes on
+				machines that have been hibernated.
+			</description>
+		</issue>
+		<issue>
+			<area>JFR</area>
```

Comment:
Shouldn't be an issue now after: https://github.com/openjdk/jmc/commit/eafd075cd4189647737ccfb90bf7643a9b43d81a